crates/extensions/c8y_mapper_ext/src/operations/firmware_update.rs

```diff
@@ -235,9 +251,17 @@ mod tests {
 
         skip_init_messages(&mut mqtt).await;
 
-        // Simulate log_upload cmd metadata message
+        // Simulate firmware_update cmd metadata message
         mqtt.send(MqttMessage::new(
             &Topic::new_unchecked("te/device/child1///cmd/firmware_update"),
+            "{}",
+        ))
+        .await
+        .expect("Send failed");
+
+        // Simulate firmware_update twin firmware message
+        mqtt.send(MqttMessage::new(
+            &Topic::new_unchecked("te/device/child1///twin/firmware"),
             r#"{"name":"firmware", "version":"1.0"}"#,
         ))
         .await
```

Comment:
Same as above. Testing `twin` can be covered in inventory. Here, just the test of receiving `te/device/child1///cmd/firmware_update` and checking if supported operation file is created is enough.